Makefile

```diff
@@ -1,38 +1,34 @@
-VERSION ?= $(shell cat VERSION)
+GITCOMMIT := $(shell git rev-parse --short HEAD 2>/dev/null)
+VERSION := $(shell git describe --tag $(GITCOMMIT) 2>/dev/null)
 
-GITCOMMIT := $(shell git rev-parse HEAD 2>/dev/null)
 GITBRANCH := $(shell git rev-parse --abbrev-ref HEAD 2>/dev/null)
 BUILDTIME := $(shell TZ=GMT date "+%Y-%m-%d_%H:%M_GMT")
-LDFLAGS := "-X main.Version=$(VERSION) -X main.GitCommit=$(GITCOMMIT) -X main.GitBranch=$(GITBRANCH) -X main.BuildTime=$(BUILDTIME)"
+LDFLAGS := "-X main.version=$(VERSION) -X main.GitCommit=$(GITCOMMIT) -X main.GitBranch=$(GITBRANCH) -X main.BuildTime=$(BUILDTIME)"
 
 SRCS = $(shell find . -name '*.go' | grep -v '^./vendor/')
 PKGS := $(foreach pkg, $(sort $(dir $(SRCS))), $(pkg))
 
 TESTARGS ?=
 
 default:
-	go build -v
+	go build -ldflags $(LDFLAGS) -v -o ./ahoy
```

Comment:
@ocean I had to move `LDFLAGS` as arguments as none of them were passed from the env variable

This is a failing CI without this change https://app.circleci.com/pipelines/github/ocean/ahoy/24/workflows/bb9e8514-6abf-4822-bde8-c9fd7dbe2850/jobs/27

Reply:
Ah of course. I had LDFLAGS as an env var feeding into the `go build...`, which I must have seen somewhere, but it doesn't work anymore I guess 🤷 

Nice pickup!